docs/usage/golang.md

```diff
@@ -40,11 +40,19 @@ Renovate will commit all files changed within the `vendor/` folder.
 
 ### Go binary version
 
-By default, Renovate will keep up with the very latest version of `go`.
+By default, Renovate will keep up with the latest version of the `go` binary.
 
-You can "pin" the `go` version that Renovate uses.
-Say you want Renovate to use Go version 1.14, you can do this by adding `go 1.14` to your `go.mod` file.
-We do not support pinning Go versions to a specific patch level, so you cannot use `go 1.14.12`, but you can use `go 1.14` in your `go.mod` file.
+You can force Renovate to use a specific version of Go by setting a constraint.
+As an example, say you want Renovate to use the latest patch version of the `1.16` Go binary, you'd put this in your Renovate config:
+
+```json
+  "constraints": {
+    "go": "1.16"
+  }
+```
```

Comment:
@rarkins Is this what you meant?

Our docs say that `constraints` is only supported for Python:

> For now this only supports `python`, other compatibility restrictions will be added in the future.

https://docs.renovatebot.com/configuration-options/#constraints

Reply:
That needs updating too! Here's what I find:

![image](https://user-images.githubusercontent.com/6311784/127054285-a9f2d687-e552-4d35-ac79-7f2105f080b8.png)

Reply:
Fixed

Reply:
@rarkins Thank you for fixing the other references to the `constraints`! 😄